pkgs/tools/audio/essentia-extractor/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl }:
+let
+  arch_table = {
+    "x86_64-linux" = "linux-x86_64";
+    "i686-linux" = "linux-i686";
+  };
+
+  sha_table = {
+    "x86_64-linux" =
+      "d9902aadac4f442992877945da2a6fe8d6ea6b0de314ca8ac0c28dc5f253f7d8";
+    "i686-linux" =
+      "46deb0a053b4910c4e68737a7b6556ff5360260c8f86652f91a0130445f5c949";
+  };
+
+  arch = arch_table.${stdenv.system};
+  sha = sha_table.${stdenv.system};
+in stdenv.mkDerivation rec {
+  pname = "essentia-extractor";
+  version = "2.1_beta2";
```

Comment:
We try to package the most up-to-date non-beta release of packages. In this case, it looks like 2.0.1 is the latest non-beta version. Could you switch to that?

Reply:
That is fair, but this is the version used and recommended by AcousticBrainz, packaging another version feels worse, somehow?

Essentia, the "parent" project has a untimely and weird release schedule where small revs (2.0 -> 2.1) actually represent _huge_ changes, and the thing is always broken anyway so a release being stable or not has little bearing on how well it works. The whole thing is weird.

Reply:
Also, it's tagged `2.1_beta2` because that's the essentia version it was built from/linked with, but the example itself has version `1.0`.

Reply:
Okay, it sounds like you have good reasons for the beta version.